pkg/corerp/frontend/controller/environments/createorupdateenvironment.go

```diff
@@ -152,10 +150,12 @@
 	resourceGroupName := rgID.FindScope("resourceGroups")
 	subscriptionID := rgID.FindScope("subscriptions")
 
-	// Ensure resource group is created.
-	err = clientv2.EnsureResourceGroupIsCreated(ctx, subscriptionID, resourceGroupName, ResourceLocation, &e.Options().Arm.ClientOptions)
+	// Get the location of the resource group
+	resourceLocation, err := clientv2.GetResourceGroupLocation(ctx, subscriptionID, resourceGroupName, &e.Options().Arm.ClientOptions)
 	if err != nil {
```

Comment:
Good change to use clientv2.GetResourceGroupLocation() instead of clientv2.EnsureResourceGroupIsCreated(). This makes the behavior more explicit about requiring an existing resource group.